pages/knowledge/aria/bad-practices/README.md

```diff
@@ -150,9 +150,3 @@ So in most cases this holds true: if you face a situation where a screen reader
 Sometimes, developers try to hide focusable elements (or elements that contain focusable children) from screen readers using `aria-hidden`. This leads to unpredictable behaviour in many screen readers, as the browser still allows to focus such elements using the `Tab` key, but because of `aria-hidden` screen readers will not announce it.
 
 If you are really curious and want to learn more about this, skip ahead and read [Hiding elements from screen readers using aria-hidden](/examples/hiding-elements/from-screen-readers).
-
-The purpose of hiding something from a specific user group leads us to the next misuse of ARIA. Read on.
-
-## Conclusion: caution with ARIA!
```

Comment:
Note to myself: I suppose we should keep a conclusion, but write something like this:

> ARIA can be the right thing to use in those rare where standard HTML does not offer a solution for an implementation requirement (for example, some advanced control patterns like tabs). Be very careful though and sure to test it thoroughly using screen readers, as not every ARIA feature is generally supported yet.

Reply:
Or the conclusion of the deleted "Good practices" page:

> There are only rare situations where HTML is not enough. In those (and only those) situations, using ARIA is advised. In other situations, when the use of a clean and semantically correct HTML structure can provide a solution, usage of ARIA is strongly discouraged. This is due to its non-homogenous support among browsers and screen readers.

Reply:
We should maybe add this to the "Conclusion" on the "Purpose" page?

Reply:
@jmuheim, I have opened https://github.com/Access4all/adg/issues/246

Reply:
👍